packages/app-mobile/root.tsx

```diff
@@ -317,6 +317,7 @@ const appReducer = (state = appDefaultState, action: any) => {
 
 				if ('smartFilterId' in action) {
 					newState.smartFilterId = action.smartFilterId;
+					newState.selectedSmartFilterId = action.smartFilterId;
```

Comment:
```suggestion
					newState.selectedSmartFilterId = action.smartFilterId;
```

A follow-up pull request (not targeting the 2.13 branch) should remove `newState.smartFilterId = ...`.